README.md

```diff
@@ -192,68 +190,30 @@ or
 
 ```
 MyEventModel = Model.extend({
-    behaviorGroup: Ember.computed(
-        'event',
-        function() {
-            return this.get( 'event' );
-        }
-    )
+    canCancel: Ember.computed( function () {
```

Comment:
We should probably add a property that this is observing, to distinguish it from just a function that returns a boolean.